code/ui/manager/src/settings/shortcuts.test.tsx

```diff
@@ -1,175 +0,0 @@
-import React from 'react';
-import { shallow } from 'enzyme';
-import { render } from '@testing-library/react';
-
-import { ThemeProvider, themes, convert } from '@storybook/theming';
-import { ShortcutsScreen } from './shortcuts';
-
-// A limited set of keys we use in this test file
-const shortcutKeys = {
-  fullScreen: ['F'],
-  togglePanel: ['A'],
-  toggleNav: ['S'],
-  toolbar: ['T'],
-  search: ['/'],
-  focusNav: ['1'],
-  focusIframe: ['2'],
-} as any;
-
-const makeActions = () => ({
-  setShortcut: jest.fn(),
-  restoreDefaultShortcut: jest.fn().mockImplementation((action) => shortcutKeys[action]),
-  restoreAllDefaultShortcuts: jest.fn().mockReturnValue(shortcutKeys),
-  onClose: jest.fn(),
-});
-
-describe('ShortcutsScreen', () => {
```

Comment:
Why delete this test?  It doesn't seem related to dependency changes.

Reply:
It's all enzyme, would love to get it replaced by something more modern, but removed it because it should already be covered (a bit) by chromatic anyway.